eppo-sdk-test/store/ConfigurationStoreTest.cs

```diff
@@ -129,13 +131,63 @@ public void ShouldClearOldValuesOnSet()
         AssertHasBandit(store, "bandit1", false);
         AssertHasBandit(store, "bandit2", false);
         AssertHasBandit(store, "bandit3", false);
-        Assert.Multiple(()=> {
+        Assert.Multiple(() =>
+        {
             Assert.That(store.TryGetMetadata("UFC_VERSION", out string? data), Is.False);
             Assert.That(data, Is.Null);
             Assert.That(store.TryGetMetadata("DICT_OBJECT", out string? storedDict), Is.False);
             Assert.That(storedDict, Is.Null);
         });
-        
+
+    }
+
+    [Test]
+    public void ShouldNotCleanBanditsIfNull()
```

Comment:
Specifically, is this testing that it shouldn't clean any models it's holding onto?

(If the UFC says we don't have bandits, then we shouldn't ever be referencing bandit models)

Reply:
This is to facilitate the case where we haven't loaded new bandits from the API. I have refactored to make this an explicit functionality of the `ConfigurationStore` rather than implied by passing `null`.